### PR TITLE
Create Tokenomics_Network_Report_Template.md

### PR DESCRIPTION
**Pull request by mochet**
Initial template

### CODE REVIEW

File: Tokenomics_Network_Report_Template.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+# Tokenomics + Network Report
+This is a report which explains the current state of the Joystream network in numbers. It pulls figures from the chain and tries to provide a basic level of information about the network, tokens and more. 
+
+## 1.0 Basic Information
+* Date Range: DD/MM/YYYY - DD/MM/YYYY
+* Council session #: XX
+* Starting block: 0
+* Block range: 12345678 - 12345678
+
+### 1.1 Block Generation Information
```
Comment (bwhm) [inline Tokenomics_Network_Report_Template.md:10]: Perhaps combine the block production and validator information in one?

Comment (bwhm) [inline Tokenomics_Network_Report_Template.md:10]: Perhaps combine the block production and validator information in one?

File: Tokenomics_Network_Report_Template.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+| Token Burn     |              |              |          |
+| USD Backing    |              |              |          |
+
+### 1.3 Membership Information
+| Property          | This Session | All Sessions | % Change |
+|-------------------|--------------|--------------|----------|
+| Number of members |              |              |          |
+
+## 2.0 Tokenomics
+### 2.1 Token generation breakdown
```
Comment (bwhm) [inline Tokenomics_Network_Report_Template.md:30]: Perhaps add Curator Role, and include burned?

File: Tokenomics_Network_Report_Template.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+### 2.2 Mints 
+| Property                    | This Session | All Sessions | % Change |
+|-----------------------------|--------------|--------------|----------|
+| Council Mint Capacity        |              |              |          |
+| Council Mint Total Minted       |              |              |          |
+| Curator Mint Capacity |              |              |          |
+| Curator Mint Total Minted |              |              |          |
+
+## 3.0 Council
```
Comment (bwhm) [inline Tokenomics_Network_Report_Template.md:46]: Consider adding some information on the outcomes of the proposals?

File: Tokenomics_Network_Report_Template.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+| Set Lead                                |                                    |                               |
+| Set Content Working Group Mint Capacity |                                    |                               |
+| Evict Storage Provider                  |                                    |                               |
+| Set Validator Count                     |                                    |                               |
+| Set Storage Role Parameters             |                                    |                               |
+* Average time for proposal vote success:
+* Average overall time for proposal vote success:
+
+## 4 Roles
+### 4.1 Validator Information
```
Comment (bwhm) [inline Tokenomics_Network_Report_Template.md:74]: As the reward here is a function of the `slotStake`, not the average stake, consider taking that into account?
Also, this role sees more slashing than others, so I think that could make sense to add as well :)

**Comment by mochet** [comment]
Thanks for the feedback!

**Comment by mochet** [comment]
This was approved by council here: https://testnet.joystream.org/#/proposals/14

**Review by bwhm — COMMENTED** [review]
Are these changes intentional? I don't understand why `LICENSE` and `README.md` should be ignored and deleted...

@mochet ?

**Comment by mochet** [comment]
> Are these changes intentional? I don't understand why `LICENSE` and `README.md` should be ignored and deleted...
> 
> @mochet ?

Sorry, made this commit some time ago and wasn't familiar with how to deal with those files. Just made changes which hopefully fix it.

**Review by bwhm — APPROVED** [review]
:+1:

**Pull request merged**